controller/src/main/java/org/jboss/as/controller/registry/ImmutableManagementResourceRegistration.java

```diff
@@ -41,10 +42,10 @@
 public interface ImmutableManagementResourceRegistration {
 
     /**
-     * A {@link RuntimePermission} needed to create a {@link ImmutableManagementResourceRegistration} or invoke one
-     * of its methods. The name of the necessary {@code RuntimePermission} is "{@code canAccessManagementResourceRegistration}."
+     * A {@link org.jboss.as.controller.security.ControllerPermission} needed to create a {@link ImmutableManagementResourceRegistration} or invoke one
+     * of its methods. The name of the permission is "{@code canAccessManagementResourceRegistration}."
      */
-    RuntimePermission ACCESS_PERMISSION = new RuntimePermission("canAccessManagementResourceRegistration");
+    ControllerPermission CAN_ACCESS_MANAGEMENT_RESOURCE_REGISTRATION = new ControllerPermission("canAccessManagementResourceRegistration");
```

Comment:
Please revert the name change. It's not necessary and this is public API. The type change is unavoidable, but it's likely anyone using this constant would only care about the "Permission"  type anyway.